vite.config.ts

```diff
@@ -28,7 +29,29 @@ const viteEjsPlugin = ({ data }: { data: ejs.Data }): Plugin => ({
   },
 });
 
+// Removes webmanifest link tag from output index.html file.
+// We add this tag if PWA features are enabled via a feature flag.
+const viteRemoveManifestPlugin = (): Plugin => ({
+  name: "Manifest",
+  enforce: "post",
+  transformIndexHtml: {
+    order: "post",
+    handler: (
+      html: string,
+      _ctx: IndexHtmlTransformContext
+    ): IndexHtmlTransformResult => {
+      return html.replace(
+        `<link rel="manifest" href="${
+          process.env.BASE_URL ?? "/"
+        }manifest.webmanifest">`,
+        ""
+      );
```

Comment:
Hmmm. This works, but feels fragile.

Reply:
I think we should assert that this changes html.